Library/Homebrew/cask/lib/hbc/cli/search.rb

```diff
@@ -15,8 +19,17 @@ def self.extract_regexp(string)
       end
 
       def self.search_remote(query)
-        matches = GitHub.search_code(user: "caskroom", path: "Casks",
-                                     filename: query, extension: "rb")
+        matches = begin
+          GitHub.search_code(
+            user: "caskroom",
+            path: "Casks",
+            filename: query,
+            extension: "rb",
+          )
+        rescue GitHub::Error => error
+          opoo "Online search failed: #{error}\n"
```

Comment:
I'd rephrase this to “Error searching on GitHub:”.

Reply:
Done in 68dd0ac